config/default/webform.webform.feedback_form.yml

```diff
@@ -64,7 +64,7 @@ elements: |-
     '#type': checkbox
     '#title': 'Data use policy'
     '#description': |-
-      Please confirm that you have read <a href="https://www-essex-gov-uk.nomensa.xyz/about-essexgovuk/privacy-and-data-protection/your-privacy">the data protection and privacy policy</a> and agree to the council using any personal data that you provide in this form to deal wtih your request.
+      Please confirm that you have read <a href="https://www-essex-gov-uk.nomensa.xyz/about-essexgovuk/privacy-and-data-protection/your-privacy">the data protection and privacy policy</a> and agree to the council using any personal data that you provide in this form to deal with your request.
```

Comment:
Also - can we change this to a relative link? `<a href="/about-essexgovuk/privacy-and-data-protection/your-privacy">` - otherwise we'll have to come back and change this when we go live - and we might forget...